### PR TITLE
Exit if run as root

### PR DESCRIPTION
**Pull request by thilobillerbeck**
Root check that warns the user and exits the application when run as root  #48

### CODE REVIEW

File: aurblobs/cli.py [hunk 1]
```diff
@@ -18,6 +18,10 @@
     except FileExistsError:
         pass
 
+if os.geteuid()==0:
```
Comment (mweinelt) [inline aurblobs/cli.py]: add spaces between operator and parameters.

File: aurblobs/cli.py [hunk 2]
```diff
@@ -18,6 +18,10 @@
...
+  print("Please don't run aurblobs as root!")
```
Comment (mweinelt) [inline aurblobs/cli.py]: use `file=sys.stderr`

File: aurblobs/cli.py [hunk 3]
```diff
@@ -18,6 +18,10 @@
...
+  exit()
```
Comment (mweinelt) [inline aurblobs/cli.py]: Defaults to zero, which would indicate "all is well", which it is not.

File: aurblobs/cli.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import os.path
 import sys
 from pathlib import Path
-
+import os
```
Comment (mweinelt) [inline aurblobs/cli.py]: Group up with the other os import.

**Comment by felixsinger** [comment]
Why not executing this check before line 15? Else the directories will be created with root privileges and won't be writeable from a non-privileged user.

**Comment by thilobillerbeck** [comment]
Checking this when im back from AFK

File: aurblobs/cli.py [hunk 5]
```diff
@@ -18,6 +18,10 @@
     except FileExistsError:
         pass
 
+if os.geteuid() == 0:
+  click.echo(click.style("Please don't run aurblobs as root!", fg='red'), file=sys.stderr)
+  sys.exit(1)
```
Comment (mweinelt) [inline aurblobs/cli.py]: Indentation is off.

**Comment by thilobillerbeck** [comment]
@felixsinger suggestion approved

**Comment by fleaz** [comment]
Thanks @mweinelt for the code review but most of your comments where also found by Travis. Imho there is no need to comment on broken indentation or pep8 violations (space around operators). This is why some smart folks invented CI pipelines in the first place ;)

**Comment by mweinelt** [comment]
What about the coloring? 
- Red vs. "Please"
- We're using no colors in other parts of the code yet

@fleaz smartass :)

**Comment by thilobillerbeck** [comment]
@mweinelt Even if the error says "please", running as root is so critical, that the color red is a god fit for it ;). I know there is no color yet, but this is something that could be changed in the future and make the output much more readable.

@fleaz CI is good, but the output is not really human readable, neither sorted (Put pep8 tests at the end). In addition to that, you don't declare anywhere that your code conforms to pep8 ;)

**Comment by thilobillerbeck** [comment]
In addition to that, it would be nice, if you put all your concerns in one comment, so I can fix up all at once and we can speed up development process. :)

**Comment by mweinelt** [comment]
I think color would be a good addition, but either we have them everywhere and consistently or not at all. So thanks for removing them at this time.

Merging.

**Comment by mweinelt** [comment]
I think color would be a good addition, but either we have them everywhere and consistently or not at all. So thanks for removing them at this time.

Squashed and merged.

**Comment by mweinelt** [comment]
Thanks, squashed and merged.

https://github.com/aurblobs/aurblobs/commit/4318bbe075bb53c2516fedce851bc48f65d2c1fd

**Comment by mweinelt** [comment]
Thanks, squashed and merged.

https://github.com/aurblobs/aurblobs/commit/4318bbe075bb53c2516fedce851bc48f65d2c1fd

**Pull request merged**